src/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wbpack Exercise</title>
+    <link  rel="stylesheet"  href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"/>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css">
+</head>
+<body>
+    <main class="container">
+        <header class="list-header"><h1>Today's To Do</h1><img class="refresh-icon" src="https://img.icons8.com/ios/50/000000/update-left-rotation.png"/></header>
+            
+        <form class='form add-list' id="add-form" action="#">
+            <input 
+                class="task-input" 
+                name="input"
+                type="text" 
+                placeholder="Add to your list..."
+                required
+                >
+                <p class="enter-key">&#8629;</p>
+        </form>
+    
+    <ul id="todo-list" class="todo-list">
+    
+    </ul>
+    <!-- 
+        <li class="list-item">
+            <div><input 
+                class="task-status"
+                type="checkbox" 
+                name="status" 
+                id="1">
+            <label class="task-desc" for="1">finish to-do list: Structure</label></div>
+            <p class="more-icon">&#8942;</p>
+        </li>
+    -->
```

Comment:
please remove the inline comments of the old pieces of code as it makes the code look messy